### PR TITLE
Feat/annoucement card

### PR DESCRIPTION
**Pull request by amphikapha**
### Type of Change

- [x] New features
- [ ] Bug fixes
- [ ] Breaking changes
- [ ] Refactor
- [ ] Documentation updates
- [ ] Other (please specify):

## Description

Annoucement Card with trimTitle for too long title

## Screenshots or GIFs (if applicable)

Already added to Playground 

Screenshot 2024-09-15 043135

### CODE REVIEW

**Comment by amphikapha** [comment]
I think each card should be clicked and redirect to its own article page, so i added the 5th prop (linkHref) to the component.

File: src/lib/components/AnnoucementCard/AnnoucementCard.svelte [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	}
+</script>
+
+<a
+	href={linkHref}
+	target="_blank"
+	class="h-[450px] w-[300px] bg-white text-sucu-gray-dark rounded shadow-card-shadow hover:shadow-card-shadow-hover transition-shadow flex flex-col items-center"
+>
+	<div class="p-3 h-[300px] w-[300px]">
+		<img src={imageURL} alt={title} width="276" height="276" class="w-[100%] rounded" />
```
Comment (punchanabu) [inline src/lib/components/AnnoucementCard/AnnoucementCard.svelte]: เพิ่ม fallback ไว้ให้หน่อย
```svelte
<div class="p-3 h-[300px] w-[300px]">
    {#if imageURL}
	<img src={imageURL} alt={title} width="276" height="276" class="w-[100%] rounded"/>
    {:else}
	<div class="w-full h-full bg-gray-300 rounded animate-pulse"></div>
    {/if}

</div>
 ```

**Review by punchanabu — CHANGES_REQUESTED** [review]
code ok เลยมีเพิ่มนิดเดียวว

**Comment by punchanabu** [comment]
อ่อแล้วก็ของ Card มันมีของ mobile size ด้วยนะ designer เพิ่งเพิ่มมา [tailwind-responsive](https://tailwindcss.com/docs/responsive-design)
Screenshot 2567-09-16 at 00 44 11

**Comment by amphikapha** [comment]
Already added fallback leaw naka

Screenshot 2024-09-19 212605


and card size for mobile

Screenshot 2024-09-19 212747

**Review by punchanabu — APPROVED** [review]

**Pull request merged**